lib/modules/neverEndingComments.js

```diff
@@ -0,0 +1,50 @@
+import * as _ from 'lodash';
+import { $ } from '../vendor';
+import { elementInViewport } from '../utils/dom';
+import * as SettingsConsole from './settingsConsole';
+
+export const module = {};
+
+module.moduleID = 'neverEndingComments';
+module.moduleName = 'Never Ending Comments';
+module.category = 'Comments';
+module.description = 'For those ultra-long comment pages, this will keep them flowing';
+module.options = {
+	loadChildComments: {
+		type: 'boolean',
+		value: false,
+		description: 'Whether or not on-screen \'load more comments\' links should be expanded when paused',
+	},
+};
+
+module.include = [
+	'comments',
+];
+
+module.go = () => {
+	window.addEventListener('scroll', _.debounce(handleScroll, 300));
+};
+
+function visibleLoaders() {
+	const loaders = module.options.loadChildComments.value ? $('span.morecomments a') : $('span.morecomments:last a');
+	return loaders.filter((i, el) => elementInViewport(el));
```

Comment:
What about eagerly breaking out of the filtering function? something like 
```javascript
let found = false;
return loaders.filter((i, el) => {
	if (found) return false;
	found = elementInViewport(el);
	return found;
});
```

not a huge fan of using the flag though...